packages/rpc-graphql/src/context.ts

```diff
@@ -2,13 +2,59 @@ import { SolanaRpcMethods } from '@solana/rpc-core';
 import { Rpc } from '@solana/rpc-transport/dist/types/json-rpc-types';
 
 import { createGraphQLCache, GraphQLCache } from './cache';
+import { AccountQueryArgs } from './schema/account/query';
 
 export interface RpcGraphQLContext {
     cache: GraphQLCache;
+    resolveAccount(args: AccountQueryArgs): ReturnType<typeof resolveAccount>;
     rpc: Rpc<SolanaRpcMethods>;
 }
 
+// Default to jsonParsed encoding if none is provided
```

Comment:
In general do you expect this layer to be more opinionated than `rpc-core`/the RPC itself, and to have defaults like this in quite a lot of places?

Reply:
In general, yes, but to what extent I'm not sure yet.

For starters, the API favors parsed data unless you specifically tell it not to parse (encoding). My thought here was basically: if you're using GraphQL, you are much more likely to be querying parsed data than encoded data, so why not make the parsed ones the default, and if you don't want to parse, you specifically say that.

I'd love your thoughts on this though.